File: C/9.2.c
/* 
9.2. Write a program in C to count the number of vowels and consonants in a string using a pointer.
*/

#include <stdio.h> 
int main() {
    char str[1000];
    int vowels = 0, consonants = 0;
    printf("Enter a string : ");
    fgets(str, sizeof(str), stdin);
    for (int i = 0; str[i] != '\0'; i++) {
        char ch = str[i];
        if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' ||
            ch == 'A' || ch == 'E' || ch == 'I' || ch == 'O' || ch == 'U') vowels++;
        else if (ch != ' ' && ((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z')))
            consonants++;
    }
    printf("Vowels : %d\n", vowels);
    printf("Consonants : %d\n", consonants);
    return 0;
}

File: Python/9.2.py
# 9.2. Write a Python program that accepts a list of integers and calculates the length and the fifth element. Return true if the length of the list is 8 and the fifth element occurs thrice in the said list.

def check_list(lst):
    if len(lst) == 8 and lst.count(lst[4]) == 3:
        return True
    return False
lst = list(map(int, input("Enter 8 integers separated by space: ").split()))
result = check_list(lst)
print("Result:", result)
